### PR TITLE
update: instruction mnemonics and archstate

### LINKED ISSUE #10: `wb_depth` not defined in ArchState
In `dma_load_mxu0`, the padding size uses `state.cfg.wb_depth`, but `ArchStateConfig` doesn't define `wb_depth` (only `wb_width` and `num_wb_registers`). This will raise `AttributeError` at runtime, and even if it existed it would likely over-pad beyond a single WB entry, causing `write_wb_u8`'s `value.numel() == wb_width/...` assert to fail. Pad to exactly one WB register's capacity (based on `wb_width`) or explicitly define/justify a config field for the intended size.

_Originally posted by @Copilot in https://github.com/ucb-ee194-tapeout/npu_model/pull/9#discussion_r2874911464_

### PR DESCRIPTION
**Pull request by T-K-233**
Each MXU has its own localized and dedicated weight buffers `w0` and `w1`.
For the `dma.load` instruction, we specify which MXU's WB to load to inside the instruction mnenonics.

To simplify the instruction mnenonics, this PR also renames `SYSTOLIC` as `mxu0` and `INNER` as `mxu1`, per the arch spec doc.

This PR also fixes #10 . Weight buffer is defined as a 1D structure from the MXU's perspective, since each cycle it can output all elements to the matmul datapath.

### CODE REVIEW

File: model_npu/configs/isa_definition.py [hunk 1]
```diff
@@ -338,10 +338,31 @@ def dma_load_m(state: ArchState, args: Dict[str, int]) -> None:
...
+    size = args["size"]
+    data = torch.tensor(state.read_memory(base, size), dtype=torch.uint8)
+    # zero pad the data to the size of the WB
+    if data.numel() < state.cfg.wb_width // torch.uint8.itemsize:
+        data = torch.nn.functional.pad(
+            data,
+            (
+                0,
+                state.cfg.wb_depth * state.cfg.wb_width // torch.uint8.itemsize
+                - data.numel(),
```
[the remark this answers is not in the document]
Comment (T-K-233) [reply model_npu/configs/isa_definition.py]: Since this is a bug from the original code, we will implement the fix in a separate PR.
